### ISSUE TITLE
Toast: Close/Dimiss button is not keyboard or screen-reader accessible

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#2420]
REF: https://modus-web-components.trimble.com/?path=/story/components-toast--default

The dismiss 'X' Should be a `<button>` (so it has tab focus) with an `aria-label` of dismiss
